packages/web/src/common/store/user-list/related-artists/sagas.ts

```diff
@@ -58,12 +53,10 @@ function* errorDispatcher(error: Error) {
 
 function* getRelatedArtists(currentPage: number, pageSize: number) {
   const id = yield* select(getId)
-  const currentUserId = yield* select(getUserId)
-  if (!id || !currentUserId) return { userIds: [], hasMore: false }
+  if (!id) return { userIds: [], hasMore: false }
```

Comment:
@Kyle-Shanks the functional fix was actually here, I just removed the user_id arg there because it was unused

Reply:
we were checking for `currentUserId` here where we shouldn't since you don't need to be logged in to get related artists

Reply:
ah I see. nice 👍